src/marker_load.c

```diff
@@ -368,10 +418,12 @@ qbool LoadBotRoutingFromFile(void)
 	fileHandle_t file = -1;
 	char lineData[128];
 	char argument[128];
+	char botFileName[128];
 
 	// Load bot definition file: frogbots rely on objects spawning 
 	//    markers, so be aware of alternative .ent files
 	char *entityFile = cvar_string("k_entityfile");
+	if (isCTF()) strlcat(entityFile, "_ctf", sizeof(entityFile));
```

Comment:
`sv_loadentfiles_dir ctf` I wonder if it's too ugly to simply use that cvar for .bot files as well. It would fix the problem, and it's only ever set to something other than `""` for CTF in the example configs, which are likely very similar to the nquakesv-configs that everyone uses.